src/shared/ui/components/BaseTable.vue

```diff
@@ -101,22 +108,22 @@ const segmentInfo = computed(() => {
 const numbers = computed(() => {
   const isMobile = width.value < PAGINATION_BREAKPOINT;
   const max = isMobile ? 6 : 10;
-  const side = isMobile ? 4 : 8;
+  const side = isMobile ? 4 : 7;
   const offset = isMobile ? 1 : 3;
```

Comment:
Too much magic constants.

Reply:
It seemed to me that from the context of the code it is clear where and what these values are used for. but you can rewrite the algorithm as you like

Reply:
It has more cognitive pressure than reading of something like:

```ts
const MOBILE_SIDE_SIZE = 4
const DESKTOP_SIDE_SIZE = 7

const side = isMobile ? MOBILE_SIDE_SIZE : DESKTOP_SIDE_SIZE
```